docs/UserGuide.adoc

```diff
@@ -523,28 +534,29 @@ Selects the 1st item in the results of the `find` command.
 *A*: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous Inventory Manager folder.
 
 == Command Summary
+// General
+* *Clear* : `clear`
+* *Help* : `help`
+* *History* : `history`
+* *Undo* : `undo`
+* *Redo* : `redo`
+// User Management
 * *Login* `login u/USERNAME p/PASSWORD`
 * *Add User* `add_user n/NAME u/USERNAME p/PASSWORD r/ROLE`
```

Comment:
@darren96 maybe consider change _ to - ?

Reply:
huh? not my pr...

Reply:
Everyone need to modify their own part, i placed comment to separate. use the format add-item add-sale etc for the commands. so @fzdy1914 may need to update your commands if you plan to have any